likeminds-chat-reactnative-integration/ChatSX/components/InputBox/index.tsx

```diff
@@ -2335,7 +2335,7 @@ const InputBox = ({
                   plainTextStyle={[
                     inputBoxStyles?.plainTextStyle,
                     {
-                      color: !isUploadScreen
+                      color: isUploadScreen
```

Comment:
why this change?

Reply:
because this is correct, earlier one was reverse